cypress/support/component.ts

```diff
@@ -5,3 +5,5 @@ import 'cypress-real-events/support'
 import './commands.jsx'
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import '@toptal/davinci-qa/src/configs/cypress/support'
+// Import style file to support Tailwind CSS classes in Cypress tests.
+import './../assets/tailwind.css'
```

Comment:
NIT: we can avoid the `./` prefix here
```suggestion
import '../assets/tailwind.css'
```

Reply:
@augustobmoura Changed, thanks!